data-privacy-note.md

```diff
@@ -0,0 +1,14 @@
+The CORD project and Community use third-party software and services
+for project management and community engagement. We are not
+responsible or liable for the privacy standards, practices, and
+policies implemented by such services.
+
+A list of currently used services and their privacy statements or
+policies is provided below to understand how they may collect
+information.
+
+* GItHub: [https://docs.github.com/en/github/site-policy/github-privacy-statement](https://docs.github.com/en/github/site-policy/github-privacy-statement)
+* YouTube: [https://policies.google.com/privacy](https://docs.github.com/en/github/site-policy/github-privacy-statement)
```

Comment:
wait, the link is all pointing to same github privacy here. Need changes in second section...

Reply:
:) copy paste error. Let me fix it right quick